internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -909,7 +911,7 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 			"private_cluster_enabled": {
 				Type:     pluginsdk.TypeBool,
 				Optional: true,
-				ForceNew: true,
+				Default:  false,
```

Comment:
@tombuildsstuff `ApiServerAccessProfile` is not returned by default anymore, which means I'm setting values by default now. Does this require this  change require a state migration?

Or is there a simpler solution?